python/mxnet/symbol.py

```diff
@@ -136,7 +154,23 @@ def __pow__(self, other):
             raise TypeError('type %s not supported' % str(type(other)))
 
     def __neg__(self):
-        """x.__neg__(y) <=> -x """
+        """x.__neg__() <=> -x
+        This function applies negative to the input, element-wise.
```

Comment:
Suggest changing to: "Numerical negative, element-wise."
This is how numpy describes it.